.gitignore

```diff
@@ -1,3 +1,6 @@
 clouds
 .ipynb_checkpoints
 __pycache__
+
+# datasets
+dataset_creation/crawler/__pycache__
```

Comment:
I would change this to: **/__pycache__/  and delete the 3rd line

Reply:
I think line 3 as it is would suffice